bin/slackin

```diff
@@ -11,7 +11,7 @@ program
 .option('-c, --channel <chan>', 'Single channel guest invite [$SLACK_CHANNEL]', process.env.SLACK_CHANNEL)
 .option('-i, --interval <int>', 'How frequently (ms) to poll Slack [$SLACK_INTERVAL or 1000]', process.env.SLACK_INTERVAL || 1000)
 .option('-s, --silent', 'Do not print out warns or errors')
-.option('-c, --css <file>', 'Full URL to a custom CSS file to use on the main page')
+.option('-c, --css <file>', 'Full URL to a custom CSS file to use on the main page', process.env.SLACKIN_CSS)
```

Comment:
I like this. We should document it in the `README` too, along the other env vars.